identity-iota/src/client/client_builder.rs

```diff
@@ -1,23 +1,25 @@
 // Copyright 2020-2021 IOTA Stiftung
 // SPDX-License-Identifier: Apache-2.0
 
-use crate::client::Client;
-use crate::client::Network;
-use crate::error::Result;
+use crate::{
+  client::{Client, Network},
+  error::Result,
+};
 
 /// A `ClientBuilder` is used to generated a customized `Client`.
-#[derive(Clone, Debug)]
```

Comment:
why this change? `bool` should be debuggable

Reply:
Opps, this is a left over change from when we briefly went with something like:
```
pub struct ClientBuilder {
  pub(crate) builder: iota::ClientBuilder,
  pub(crate) network: Network,
}
``` 
before noticing that it's probably better not to wrap it in the struct. Reverted this